scripts/x-date-pickers-pro.exports.json

```diff
@@ -34,10 +34,10 @@
   { "name": "DateCalendarSlotProps", "kind": "Interface" },
   { "name": "DateCalendarSlots", "kind": "Interface" },
   { "name": "DateField", "kind": "Variable" },
-  { "name": "DateFieldInPickerProps", "kind": "TypeAlias" },
   { "name": "DateFieldProps", "kind": "TypeAlias" },
   { "name": "DateOrTimeView", "kind": "TypeAlias" },
   { "name": "DatePicker", "kind": "Variable" },
+  { "name": "DatePickerFieldProps", "kind": "TypeAlias" },
```

Comment:
That's not super clean, but I'm proposing not to advertise that we are doing super small BCs in a minor :laughing: 

How would you add it to the changelog?
A highlight adds a lot of noise for a super small info and having a highlight with a BC in a minor is not a great communication :grimacing: 

Do you want me to extract this logic in a standalone PR so that it's listed in the changelog?

Reply:
> That's not super clean, but I'm proposing not to advertise that we are doing super small BCs in a minor 😆

> How would you add it to the changelog?
A highlight adds a lot of noise for a super small info and having a highlight with a BC in a minor is not a great communication 😬

Good points. 👍 
We can keep such small changes from highlights. 👍 

> Do you want me to extract this logic in a standalone PR so that it's listed in the changelog?

This could be a nice compromise. Bringing a slight visibility to the ones, who might care. 👍